service/worker/batcher/batcher.go

```diff
@@ -76,6 +78,7 @@ func (s *Batcher) Start() error {
 	ctx := headers.SetCallerInfo(context.Background(), headers.SystemBackgroundCallerInfo)
 	workerOpts := worker.Options{
 		BackgroundActivityContext: ctx,
+		Identity:                  fmt.Sprintf("temporal-system@%d", os.Getpid()),
```

Comment:
This isn't unique across workers (everyone is pid 1 in their container).

We can throw in the hostname too, or just use `temporal-system@<random uuid>`?

Reply:
Host name would be easier to debug.